src/github.com/getlantern/lantern-android/client/client.go

```diff
@@ -59,7 +52,7 @@ func NewClient(addr, appName string) *MobileClient {
 
 	hqfd := client.Configure(clientConfig.Client)
 
-	hqfdc := hqfd.DirectHttpClient()
+	hqfdc := hqfd.NewDirectDomainFronter()
```

Comment:
I just created https://github.com/getlantern/lantern/issues/2540. I'm happy to just knock that out unless you're itching to get to it.